clang-tools-extra/clang-tidy/utils/RenamerClangTidyCheck.cpp

```diff
@@ -61,6 +61,44 @@ struct DenseMapInfo<clang::tidy::RenamerClangTidyCheck::NamingCheckId> {
 namespace clang::tidy {
 
 namespace {
+/// Returns the function that \p Method is overridding. If There are none or
+/// multiple overrides it returns nullptr. If the overridden function itself is
+/// overridding then it will recurse up to find the first decl of the function.
+const CXXMethodDecl *getOverrideMethod(const CXXMethodDecl *Method) {
```

Comment:
anyonymous namespace -> static (per llvm codding guidelines)
leave only NameLookup in anonymous namespace

Reply:
Ah. Good to know.